docs/pl/mkdocs.yml

```diff
@@ -52,6 +52,8 @@ nav:
   - tr: /tr/
   - uk: /uk/
   - zh: /zh/
+- Tutorial - User Guide:
+  - tutorial/index.md
```

Comment:
And this is something I would **Definitely** change to 
```yml
- Samouczek:
  - tutorial/index.md
```
We should avoid naming this topic in English, its much better to use Polish here. FastAPI docs are smart enough to fill docs we don't cover in English even if we use Polish name (screenshot), this is probably detected by `tutorial/index.md`.

See current screenshot (I pulled your forked repo and changed code like above).


![image](https://user-images.githubusercontent.com/51059348/153054754-7be7e767-da9c-4ec8-96a7-a6dd2eeffe14.png)